castanet/counter.py

```diff
@@ -43,7 +43,14 @@ def count_whileloops(cast_dict):
 
 
 def count_forloops(cast_dict):
-    """A function for counting the number of while loops in a Python program."""
+    """A function for counting the number of while loops in a Python program.
+
+    Args:
+        cast_dict: counts the number of for loops in a file
+
+    Returns:
+        int: total number of for loops in a file
```

Comment:
This function returns dictionary not an int. This should be changed to `dict: files and the corresponding for loop amounts` or something similar